.github/workflows/podvm.yaml

```diff
@@ -55,12 +55,31 @@ jobs:
         username: ${{ github.repository_owner }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Determine image tags
+      run: |
+        event_name=${{ github.event_name }}
+        img="${{ inputs.registry }}/podvm-${{ matrix.provider }}-${{ matrix.os }}-${{ matrix.arch }}"
+        tags=""
+
+        case "$event_name" in
+        release)
+          tags="${img}:latest,${img}:${{ github.sha }},${img}:${{ github.release.tag_name }}" ;;
+        pull_request|pull_request_target)
+          tags="${img}:ci-pr${{ github.event.number }}" ;;
+        push|workflow_dispatch)
+          tags="${img}:latest,${img}:${{ github.sha }}" ;;
```

Comment:
workflow_dispatch is manual executing no? do we want to push to latest i this case?

Reply:
@snir911 yes, `workflow_dispatch` is manually executing. With current workflows we some times (very rare I'd say) manually build the images from main and they should get labeled `latest`